content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,122 @@
+
+### Key features
+
+- Peer connections: WebRTC enables
+  [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes.
+
+- Data channels: WebRTC provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send a text or binary data over an active connection to a peer.
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/)) to connect to nodes that run behind
+  NATs and firewalls. In non-decentralized WebRTC, this can be facilitated by a
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
+  Using a custom signaling protocol or a different signaling service is also possible.
+  Overall, this allows for faster and more efficient communication.
+
+- Security: WebRTC connections are encrypted using
+  [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security).
+
+Browsers expose an API to establish WebRTC connections. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate.
+
+### Browser-to-Server
+
+libp2p WebRTC enables browsers to connect to public server nodes without those
+endpoints providing a TLS certificate within the browser's trustchain.
+
+{{< alert icon="" context="info" text="When connecting to a WebSocket server, browsers require the server to present a TLS certificate signed by a trusted CA (certificate authority). Few nodes have such a certificate, which is the reason that WebSocket never saw widespread adoption in the libp2p network. WebRTC and WebTransport support encrypted communication without requiring a signed certificate from a trusted CA." />}}
+
+In libp2p:
+
+- WebRTC multiaddresses are composed of a standard UDP multiaddr,
+  followed by `webrtc` and the `multihash` of the certificate that
+  the node uses, as such:
+  `/ip4/1.2.3.4/udp/1234/webrtc/certhash/<hash>/p2p/<peer-id>`;
+- WebRTC encrypts connections using DTLS. However, an additional handshake is required to
+  authenticate a peer's peer ID once the WebRTC connection has been established.
+
+A browser can connect to a server node without needing a trusted TLS
+certificate. View [this scenario](https://github.com/libp2p/specs/blob/master/webrtc/README.md#browser-to-public-server) on browser-to-server connection establishment
+in the technical specification.
+
+Contrary to the standard WebRTC handshake process, the browser and server do not
```

Comment:
```suggestion
To understand the standard WebRTC connection process and how it differs in libp2p, let's begin by familiarizing ourselves with some internal concepts.
The first is the Session Description Protocol (SDP). The SDP is a protocol that is used to describe multimedia sessions. SDP offer and answer messages are exchanged by clients and servers to establish sessions.

Contrary to the standard WebRTC handshake process, the libp2p browser and server nodes do not
```
I think this paragraph needs a bit of introduction.

Reply:
This was the initial approach for the doc (that is, introduce key concepts), but was later simplified to focus on an abstraction of the process. I'm a fan of the flow you proposed in the suggestions. 

However, we'd have to introduce multiple concepts, and to [Marten's previous point](https://github.com/libp2p/docs/pull/264#discussion_r1084765199), we should explain others (like ICE and STUN, which, imo, are just as important).

We can do this, the approach right now was to link out to external content.

Then again, that would mean the [alert] section on SDP could be removed.

Reply:
Here is what I'm thinking and this is in response to the suggestions for introduction SDP and DTLS (just utilizing the features list and focusing on the process under "WebRTC in libp2p"): https://github.com/libp2p/docs/pull/264/commits/334eca2b07020ca440e6fa481e9244d35306de33

Reply:
Yeah I think that makes sense. I'd simply the sentence to though:
>  [SDP (Session Description Protocol)](https://en.wikipedia.org/wiki/Session_Description_Protocol) to negotiate the parameters
  of the data channel and other properties.